tests/data_files/ignored.lst

```diff
@@ -0,0 +1,5 @@
+^ M
```

Comment:
what is this "^M" for?

Reply:
`git status -s` command will output ` M`(modified file) , ` D` (deleted file) and `??`(untracked file). 
Modified files are ignored in the tests